### PR TITLE
Add chartjs 2.4 support

### PR DESCRIPTION
**Pull request by FilipeDominguesGit**
Added support to chart.js 2.4 and updated Destroy and its test.

### CODE REVIEW

**Comment by sedouard** [comment]
Thanks @FilipeDominguesGit! I'll prioritize merging this one over #9 since the solution is originally yours. If you can address review comments and I'll get the changes merged and published straight away. Thanks! 🍻🍻

**Comment by kdekooter** [comment]
I would suggest setting the peer dependency to `<=2.4.*` in order to prevent similar trouble in the future.

**Comment by sedouard** [comment]
I agree with that. I can change the peer dependency & docker file to use <=2.4.* or you can include it in the PR.

**Comment by kdekooter** [comment]
I just closed my PR in favour of this one 😉

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ RUN cd $(npm root -g)/npm \
   && npm install fs-extra \
   && sed -i -e s/graceful-fs/fs-extra/ -e s/fs\.rename/fs.move/ ./lib/utils/rename.js
 RUN npm install -g grunt-cli mocha-cli
-RUN npm install chart.js@2.3.0
+RUN npm install chart.js
```
Comment (FilipeDominguesGit) [inline Dockerfile]: I've changed it to `RUN npm install chart.js@"<=2.4.*"` or would you prefer `RUN npm install chart.js@2.*.*` ?

File: index.js [hunk 2]
```diff
@@ -37,59 +36,76 @@ class ChartjsNode {
...
+                Object.keys(window).forEach(property => {
+                    if (typeof global[property] === 'undefined') {
+                        global[property] = window[property];
+                        this._windowPropertiesToDestroy.push(property);
+                    }
+                });
+
+                // adding all window.HTMLCanvasElement methods to global.HTMLCanvasElement
+                canvasMethods.forEach(method =>
+                    global[method] = window[method]
```
Comment (sedouard) [inline index.js]: actually I suppose this is necessary whether it is defined or not. so fine for me to leave the check out.

**Comment by sedouard** [comment]
LGTM! thanks @FilipeDominguesGit! 🍻

**Pull request merged**